.github/issue_template/03_mybuildisbroken.md

```diff
@@ -0,0 +1,46 @@
+---
+name: 😵☠🐱‍💻👩‍💻💔💢💥❌🚫❗❕ (Listed out emojis that fit here, need to choose one) My Build is Broken
```

Comment:
😄 
I liked emojis 0, 5, and 6 best. 2 is nice but I don't see anything that indicates "broken" to me.